mongodbatlas/resource_mongodbatlas_container.go

```diff
@@ -88,6 +88,11 @@ func resourceContainerCreate(d *schema.ResourceData, meta interface{}) error {
 		params.NetworkName = d.Get("network_name").(string)
 	}
 
+	if params.ProviderName == "AWS" {
+		params.RegionName = d.Get("region").(string)
+		params.AtlasCidrBlock = d.Get("atlas_cidr_block").(string)
```

Comment:
Don't think this line is needed since we already set the CIDR block in the params above- https://github.com/akshaykarle/terraform-provider-mongodbatlas/blob/18a6516f7812b4a6efc26c1edb0e4d620893111a/mongodbatlas/resource_mongodbatlas_container.go#L81

Reply:
will merge and remove this extra line since I haven't heard back yet